tools/logger/convert.c

```diff
@@ -1,16 +1,13 @@
-/*
- * debug log converter, using new logger format.
+/* SPDX-License-Identifier: BSD-3-Clause
```

Comment:
Convention is to have `//` comments in .c files and `/**/` in .h. Examples like in this post: https://github.com/thesofproject/sof/issues/1497#issuecomment-497619107

Reply:
Sorry, I didn't notice there was any difference in notation. Fixed.